pegasus/sites.v3/code.org/public/playlab.haml

```diff
@@ -5,14 +5,14 @@ social:
   "twitter:card": photo
   "twitter:site": "@codeorg"
   "twitter:url": "http://code.org/playlab"
-  "twitter:image:src" : "https://code.org/images/playlab_original3.png"
+  "twitter:image:src" : "https://code.org/shared/images/courses/logo_tall_playlab.jpg"
   "twitter:title": "Play Lab Tutorials"
   "twitter:description": "Select between Gumball, Ice Age, and Classic themes to create your own story or game."
   "og:title": "Play Lab Tutorials"
   "og:description": "Select between Gumball, Ice Age, and Classic themes to create your own story or game."
-  "og:image" : "https://code.org/images/playlab_original3.png"
-  "og:image:width": "400"
-  "og:image:height": "400"
```

Comment:
does twitter work okay with this dimension change?  any chance 400x400 was chosen specially?  i'd be inclined to use same dimensions if we're not sure.